<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadVideoDialog</class>
 <widget class="QDialog" name="UploadVideoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>录制与上传</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_record">
      <attribute name="title">
       <string>录制并上传</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_record">
       <item>
        <widget class="QLabel" name="label_recordPreview_title">
         <property name="text">
          <string>录制预览</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_recordPreview">
         <property name="minimumSize">
          <size>
           <width>640</width>
           <height>360</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>640</width>
           <height>360</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="styleSheet">
          <string>background-color: black;</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_record_buttons">
         <item>
          <widget class="QPushButton" name="pb_startRecord">
           <property name="text">
            <string>开始录制</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_stopRecord">
           <property name="text">
            <string>停止录制</string>
           </property>
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_record">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_record_path">
         <item>
          <widget class="QLabel" name="label_recordPath_title">
           <property name="text">
            <string>录制文件路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_recordPath">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_record">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_record_upload">
         <item>
          <spacer name="horizontalSpacer_record_upload">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_uploadRecorded">
           <property name="text">
            <string>上传录制的视频</string>
           </property>
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_upload">
      <attribute name="title">
       <string>上传本地视频</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_upload">
       <item>
        <widget class="QLabel" name="label_restriction">
         <property name="text">
          <string>上传本地文件只支持文件大小不超过50MB,格式为FLV和MP4的文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_advice">
         <property name="text">
          <string>为了增加视频热度请在上传前,为视频添加标签,以便用户分类及搜索</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_file">
         <item>
          <widget class="QLineEdit" name="le_filePath">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_browse">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_content">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_thumbnail">
           <item>
            <widget class="QLabel" name="label_thumbnail_title">
             <property name="text">
              <string>缩略图</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_thumbnail">
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>300</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>300</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="styleSheet">
              <string>background-color: black;</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_categories">
           <item>
            <widget class="QLabel" name="label_categories_title">
             <property name="text">
              <string>视频分类</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_food">
             <property name="text">
              <string>美食</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_funny">
             <property name="text">
              <string>搞笑</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_positive">
             <property name="text">
              <string>正能量</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_dance">
             <property name="text">
              <string>舞蹈</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_song">
             <property name="text">
              <string>歌曲</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_video">
             <property name="text">
              <string>影视</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_outside">
             <property name="text">
              <string>户外</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_education">
             <property name="text">
              <string>教育</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_categories">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_progress">
         <item>
          <widget class="QLabel" name="label_progress_title">
           <property name="text">
            <string>上传进度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_progress_percent">
           <property name="text">
            <string>0%</string>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_progress">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_startUpload">
           <property name="text">
            <string>开始上传</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

